static/js/search_suggestion.js

```diff
@@ -249,11 +247,20 @@ function get_person_suggestions(people_getter, last, operators, autocomplete_ope
 
     const prefix = Filter.operator_to_prefix(autocomplete_operator, last.negated);
 
-    const highlight_person = make_person_highlighter(query);
-
     const objs = persons.map((person) => {
-        const name = highlight_person(person);
-        const description = prefix + " " + name;
+        const avatar_url = people.small_avatar_url_for_person(person);
+        const get_display_name = () => {
+            if (settings_data.show_email()) {
+                return `${person.full_name} &lt;${person.email}&gt;`;
```

Comment:
What do this look like for Desdemona?  (I think she has permission to see emails; if not edit organization settings).

I think it's likely we'll prefer to just have the pills (with no email address in the text), in which case we may not need this code.